src/track/serato/markers.cpp

```diff
@@ -29,14 +32,14 @@ const quint16 kVersion = 0x0205;
 // See this for details:
 // https://github.com/Holzhaus/serato-tags/blob/master/docs/serato_markers_.md#color-format
 
-QRgb seratoColorToRgb(quint8 w, quint8 x, quint8 y, quint8 z) {
+mixxx::RgbColor seratoColorToRgb(quint8 w, quint8 x, quint8 y, quint8 z) {
     quint8 b = (z & 0x7F) | ((y & 0x01) << 7);
     quint8 g = ((y & 0x7F) >> 1) | ((x & 0x03) << 6);
     quint8 r = ((x & 0x7F) >> 2) | ((w & 0x07) << 5);
-    return QRgb(r << 16) | (g << 8) | b;
+    return mixxx::RgbColor((r << 16) | (g << 8) | b);
```

Comment:
As far as I remember the return type of << is implementation defined, but at least the type of the left hand operator. 
To make this save, please shift a 32 bit uint.

Reply:
https://en.cppreference.com/w/cpp/language/operator_arithmetic